openmls/Cargo.toml

```diff
@@ -49,7 +49,7 @@ test-utils = [
 ]
 crypto-debug = [] # ☣️ Enable logging of sensitive cryptographic information
 content-debug = [] # ☣️ Enable logging of sensitive message content
-js = ["dep:getrandom"] # enable js randomness source for provider
+js = ["dep:getrandom", "dep:fluvio-wasm-timer"] # enable js randomness source for provider
```

Comment:
Should these really go here or are they only needed for tests?
Generally we should get rid of the `getrandom` dependency like this here.
See https://docs.rs/getrandom/latest/getrandom/#webassembly-support

But this is for a follow-up.

Reply:
So yes, `getrandom` does it nicely by requiring you to set the js flag, but `fluvio-wasm-timer` just assumes that the JS environment is available when you are running in `wasm32-unknown-unknown`. In order to gracefully fail in the situation that someone tries to use OpenMLS on `wasm32-unknown-unknown` without a JS env, I used the same approach as `getrandom`, because that seems the sanest. Otherwise, the caller only knows they need a JS env for randomness, not timing.

Though I suppose that if randomness works, timing likely will as well. Maybe I was overthinking here, but I wasn't super confident with just using the JS env when we are not sure that it is available.